jpa/eclipselink.jpa.test.jse/src/it/java/org/eclipse/persistence/jpa/test/storedproc/TestStoredProcedures.java

```diff
@@ -201,7 +205,7 @@ private static boolean createSimpleStoredProcedure(EntityManagerFactory emf) {
             //Add more platform specific diction to support more platforms
             if(platform.isMySQL()) {
                 proc.addStatement("SET out_param_one = CONCAT('One: ',in_param_one,' Two: ',in_param_two,' Three: ',in_param_three)");
-            } else if(platform.isOracle()) {
+            } else if(platform.isOracle() || platform.isPostgreSQL()) {
```

Comment:
It throws

```
[ERROR]   TestStoredProcedures.setup:54->createSimpleStoredProcedure:222 » Database 
Internal Exception: org.postgresql.util.PSQLException: ERROR: procedures cannot have OUT arguments
  Hint: INOUT arguments are permitted.
Error Code: 0
Call: CREATE PROCEDURE simple_order_procedure (
	in_param_one VARCHAR(10),
	in_param_two VARCHAR(10),
	in_param_three VARCHAR(10),
	 OUT out_param_one VARCHAR(30)) AS
.....
```

Reply:
Apparently PG only supports OUT params since version 14. I have 17 installed, so it worked for me. I've modified the test to use INOUT instead for PG. It works on 17, and hopefully it works on whatever version you're testing with.